README.md

```diff
@@ -230,6 +235,12 @@ to authenticate and retrieve a session cookie. This cookie can be fed
 to openfortivpn using option `--cookie-on-stdin`. Obviously, such a
 solution requires a graphic session.
 
+When started using `--saml-login` the program creates a web server that
+accepts SAML login requests. To login using SAML you just have to open
+`<your-vpn-domain>/remote/saml/start?redirect=1` and follow the login steps.
+At the end of the login process the page will be redirected to 
```

Comment:
I don't think it's sufficient to put this into the README. It should also be in the manpage, and it might be a good idea to print a link when the program runs. That way, the user can simply click the link in the terminal.

Reply:
I've added an extra commit to print the actual URL with `log_info` to the terminal already.
But still I'll add this to the man page as well.